app/models/katello/repository.rb

```diff
@@ -992,6 +993,8 @@ def index_content(options = {})
       # some times we want to force fetch all data even for non-library repos.
       # Use the full_index for those times
 
+      root.update!(allow_updates: true) if root.is_container_push?
```

Comment:
Same comment as above. Now that I've commented the same thing thrice, maybe this would be better as a method on `root`, too - a method that does the `root.update_columns` for you. and maybe call it `set_allow_updates`

Reply:
Same as above. Thanks for the thoughts.